pkg/pillar/cmd/volumemgr/handlecontent.go

```diff
@@ -148,16 +148,27 @@ func lookupContentTreeStatusAny(ctx *volumemgrContext, key string) *types.Conten
 	return nil
 }
 
-func getAllAppContentTreeStatus(ctx *volumemgrContext) map[string]*types.ContentTreeStatus {
-	log.Tracef("getAllAppContentTreeStatus")
+func getAllContentTreeStatus(ctx *volumemgrContext) map[string]*types.ContentTreeStatus {
+	log.Tracef("getAllContentTreeStatus")
+	contentIDAndContentTreeStatus := make(map[string]*types.ContentTreeStatus)
+
+	//Getting all AppImgObj ContentTreeStatus
```

Comment:
We have other functions in volumemgr which have an outer loop
	for _, objType := range ctObjTypes {

I think it makes sense to do the same thing here.

Reply:
I push an empty commit by mistake and that closed this PR.
I addressed this comment as part of https://github.com/lf-edge/eve/pull/1716.